packages/snaps-controllers/package.json

```diff
@@ -135,5 +136,17 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "lavamoat": {
+    "allowScripts": {
```

Comment:
I think these can be removed. AFAIK we only need to specify these at the root package.json